### ISSUE TITLE
0.9.14a : Faction reward

### ISSUE THREAD

**Issue by SirLagardere** [brittyazel/Neuron#32]
Hi,
I don't know if the "Reward" text should appeared when you've got the Bonus Faction package, but if it sould, it's not "working" on my side : 
![image](https://user-images.githubusercontent.com/20615278/34735828-15f9e6a0-f571-11e7-9293-6610f0af61af.png)

**Comment by brittyazel** [comment]
Yeah it should. Apparently it's not working heh. I'll look at it tonight

**Comment by brittyazel** [comment]
Strange I don't know why it's not showing. The code looks correct. I didn't t have any rewards yet so I've yet to see it myself.

**Comment by SirLagardere** [comment]
So, on my last reward, the text appears : 
![image](https://user-images.githubusercontent.com/20615278/34907837-04a52e7a-f885-11e7-901d-d1d3c02da5dd.png)
But, as I reclaim it, the text persists until a new reputation gain (maybe it's a bug on Blizzard side)

**Comment by brittyazel** [comment]
Is this still an issue? I never followed up to see if my fix worked

**Comment by SirLagardere** [comment]
I need to check it carrefully, but as far as I know, there still a problem. It seems to be updated only after another gain. So for example if I can get a reward, it will only be updated when I gain reputation again.
And when I have the alert and collect the reward, it's still marked on the bar.

**Comment by SirLagardere** [comment]
Another thing, the faction rank is not translated (ie exalted stay exalted, not localized)

**Comment by SirLagardere** [comment]
So as far as I know the problem is still there. Don't know if it's a problem on Blizzard side or not but the reward highlight is always delayed by one reputation gain (ie : reward available, reputation gain, highlight on, reward claimed, reputation gain, highlight off)

**Comment by brittyazel** [comment]
lol ok I'll finally figure this one out. I'll go out and farm some world quests until I get the reward, I've been putting off doing world quests for a long time lol

**Comment by brittyazel** [comment]
Ok so the issue is that the text that says "reward" won't change until a /reload. I just need to add a check to some sort of onevent call
